python/versions.bzl

```diff
@@ -34,6 +34,7 @@ TOOL_VERSIONS = {
             "x86_64-apple-darwin": "8d06bec08db8cdd0f64f4f05ee892cf2fcbc58cfb1dd69da2caab78fac420238",
             "x86_64-unknown-linux-gnu": "aec8c4c53373b90be7e2131093caa26063be6d9d826f599c935c0e1042af3355",
         },
+        "strip_prefix": "python",
```

Comment:
curious, are these required here? or are you just being explicit by setting this to the default value?

Reply:
It's kinda both. There's no default so without this the toolchain would not be structured correctly. So to continue to have these particular ones function correctly, I added the strip_prefixes. I kinda like having these here since I noticed the archive structure changed some time ago. Looking at https://github.com/indygreg/python-build-standalone/releases/tag/20200822, the toolchains are in a `python/install` directory. My hope in adding these here is that it'd draw attention to the fact that this might be needed to get toolchains working.